src/games/include/OpenKneeboard/DCSWorld.h

```diff
@@ -41,10 +41,10 @@ class DCSWorld final : public OpenKneeboard::Game {
   static std::filesystem::path GetSavedGamesPath(Version);
   virtual bool MatchesPath(const std::filesystem::path&) const override;
 
-	virtual std::shared_ptr<GameInstance> CreateGameInstance(
-		const std::filesystem::path&) override;
-	virtual std::shared_ptr<GameInstance> CreateGameInstance(
-		const nlohmann::json&) override;
+  virtual std::shared_ptr<GameInstance> CreateGameInstance(
+    const std::filesystem::path&) override;
+  virtual std::shared_ptr<GameInstance> CreateGameInstance(
+    const nlohmann::json&) override;
```

Comment:
I'll just run clang-format over everything again in a separate commit, though I think EVT_AIRCRAFT can be used here so a change isn't needed